File: Web2/src/app/types/user/create-user-request.ts
export interface CreateUserRequest {
    createdDate: Date;
    createdBy: string;
    updatedDate: Date;
    updatedBy: string;
    firstName: string;
    middleName: string;
    lastName: string;
    email: string;
    phone: number;
    passwordHash: string;
    passwordSalt: string;
    isActive: boolean;
  }

File: Scylla/UserManagement/Models/User/CreateUserRequest.cs
using Shared.Models.Address;
using System.ComponentModel.DataAnnotations;

namespace UserManagement.Models.User
{
    public class CreateUserRequest
    {

        [Required]
        public string Email { get; set; }

        [Required]
        public string FirstName { get; set; }

        public string MiddleName { get; set; }

        public int Phone { get; set; }

        [Required]
        public string? LastName { get; set; }

        [Required]
        public string Password { get; set; }

        public List<CreateAddressRequest> AddressList { get; set; }

    }

}
